src/components/Avatar/Avatar.stories.tsx

```diff
@@ -1,18 +1,17 @@
 import { BADGE } from '@geometricpanda/storybook-addon-badges';
-import { Story, Meta } from '@storybook/react';
+import { StoryObj } from '@storybook/react';
 import React from 'react';
 
-import { Avatar, Props } from './Avatar';
+import { Avatar } from './Avatar';
 
 export default {
   title: 'Example/Avatar',
   component: Avatar,
   parameters: {
     badges: [BADGE.BETA],
   },
-} as Meta;
```

Comment:
`as Meta<Args>` provides more typing to support `decorators`, and not sure what else. Having `as Meta<Args>` types the `Story` in the argument. Think everywhere else that's using `decorators` is using `Meta<Args>` as well (such as in `Text.stories.tsx`).

Being said, I don't think adding `Meta<Args>` to every story file hurts, just didn't want to import it if not being used.

![image](https://user-images.githubusercontent.com/86632227/173685065-de337823-b99c-440d-831b-1d30b19e72d7.png)

Reply:
Thanks for the explanation. Now I'm wondering if we should leave it on all of them just because that makes it less likely we'll forget it on components where it is useful.